br/backup-and-restore-faq.md

```diff
@@ -41,21 +41,29 @@ Running BR with the root access might fail due to the disk permission, because t
 ## What should I do to handle the `Io(Os...)` error?
 
 Almost all of these problems are system call errors that occur when TiKV writes data to the disk. You can check the mounting method and the file system of the backup directory, and try to back up data to another folder or another hard disk.
- 
+
 For example, you might encounter the `Code: 22(invalid argument)` error when backing up data to the network disk built by `samba`.
 
 ## What should I do to handle the `rpc error: code = Unavailable desc =...` error occurred in BR?
- 
+
 This error might occur when the capacity of the cluster to restore (using BR) is insufficient. You can further confirm the cause by checking the monitoring metrics of this cluster or the TiKV log.
 
 To handle this issue, you can try to scale out the cluster resources, reduce the concurrency during restore, and enable the `RATE_LIMIT` option.
 
 ## Where are the backed up files stored when I use `local` storage?
- 
+
 When you use `local` storage, `backupmeta` is generated on the node where BR is running, and backup files are generated on the Leader nodes of each Region.
 
 ## How about the size of the backup data? Are there replicas of the backup?
 
 During data backup, backup files are generated on the Leader nodes of each Region. The size of the backup is equal to the data size, with no redundant replicas. Therefore, the total data size is approximately the total number of TiKV data divided by the number of replicas.
 
 However, if you want to restore data from local storage, the number of replicas is equal to that of the TiKV nodes, because each TiKV must have access to all backup files.
+
+## What should I pay attention to when BR restores data to the upstream cluster of TiCDC/Drainer?
+
++ **The data restored by BR cannot be replicated to the downstream**. Because BR directly imports SST files, and the downstream cluster currently cannot obtain upstream SST files.
```

Comment:
```suggestion
+ **The data restored using BR cannot be replicated to the downstream**. This is because BR directly imports SST files but the downstream cluster currently cannot obtain these files from the upstream.
```